docs: how to configure the mediator (#60)
input-output-hk/atala-prism-mediator: Coordinate-Mediation-Protocol.md

# Coordinate Mediation Protocol

> This Protocol is parte of the **DIDComm.org**

Reviewer: ```suggestion
This protocol is a part of the [DIDComm](https://didcomm.org/).
```

> Is a protocol to coordinate mediation configuration between a mediating agent and the recipient.

Reviewer: ```suggestion
It coordinates the mediation configuration between a mediating agent and the recipient.
```

This protocol follows the request-response message exchange pattern, and only requerires the simple state of waiting for a response or to produce a response.

> See [https://didcomm.org/mediator-coordination/2.0/]

Reviewer: ```suggestion
For a detailed description of the protocol, please, see [Mediator Coordination v2.0](https://didcomm.org/mediator-coordination/2.0/).
```
